chat-plugins/mafia.js

```diff
@@ -354,6 +354,7 @@ class MafiaTracker extends Rooms.RoomGame {
 		if (!player && this.dead[user.userid] && this.dead[user.userid].restless) player = this.dead[user.userid];
 		if (!(target in this.players) && target !== 'nolynch') return false;
 		if (player.lynching || (target === player.userid && !this.selfEnabled)) return false;
+		if (target === player.userid && (this.lynches[target] && this.getHammer() - 1 > this.lynches[target].count || 0) && this.selfEnabled === 'hammer') return false;
```

Comment:
``count || 0`` is pointless since ``this.lynches[target]`` is guaranteed to exist and have a proper value at this point